README.md

```diff
@@ -58,8 +58,204 @@ document.addEventListener('DOMContentLoaded', function() {
 })
 ```
 
-===============================================================================================
+# Example 3: tsneez visualization
+(a) Create a HTML document to view the visualization. The HTML script elements of this document embeds jquery, d3, randomColor and tsneez javascript files. The style element of this doucment contains style information which is applied to the tsneez visualization.
 
-Fast javascript implementation of T-SNE with tree-based acceleration
+```javascript
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+        <script src="/tsneez/bower_components/d3/d3.min.js"></script>
+        <script src="/tsneez/bower_components/randomcolor/randomColor.js"></script>
+        
+        <script src="/tsneez/dist/tsneez.js"></script>
+        <script src='./example3.js'></script>
+        
+        <title>tsneez Visualization</title>
+        
+        <style>
+            .viewport {
+                width: 800px;
+                height: 600px;
+            }
+            .viewport svg {
+                width: 800px;
+                height: 600px;
+                display: block;
+            }
+            .viewport svg text {
+                pointer-events: none;
+                font-family: 'Lato', Arial;
+                font-weight: 500;
+                font-size: 1em;
+                fill: #444;
+            }
+            .infoCard {
+                position: relative;
+            }
+            .infoCard .infoBody {
+                position: absolute;
+                top: 0px;
+            }
+        </style>
+        
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="infoCard">
```

Comment:
no need for infoCard

Reply:
Ok. 
Added style for "container" class.